### ISSUE TITLE
C128 extra-keys keyboard support

### ISSUE THREAD

**Issue by Kroc** [Kroc/elite-harmless#55]
Add support for the extra keys on the C128 keyboard (such as the cursor keys), details here:
http://codebase64.org/doku.php?id=magazines:chacking6#three-key_rollover_for_the_c-128_and_c-64

**Comment by polluks** [comment]
Is there any relation to [Elite 128 2.0](https://csdb.dk/release/?id=185931)?

**Comment by Kroc** [comment]
No, sorry; Uz has never released his code (I asked), so I had to start from scratch.
